exporters/otlp/profiles/src/main/java/io/opentelemetry/exporter/otlp/profiles/LineMarshaler.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.profiles;
+
+import io.opentelemetry.exporter.internal.marshal.MarshalerUtil;
+import io.opentelemetry.exporter.internal.marshal.MarshalerWithSize;
+import io.opentelemetry.exporter.internal.marshal.Serializer;
+import io.opentelemetry.proto.profiles.v1experimental.internal.Line;
+import java.io.IOException;
+import java.util.List;
+import java.util.function.Consumer;
+
+final class LineMarshaler extends MarshalerWithSize {
+
+  private static final LineMarshaler[] EMPTY_REPEATED = new LineMarshaler[0];
+
+  private final long functionIndex;
+  private final long line;
+  private final long column;
+
+  static LineMarshaler create(LineData lineData) {
+    LineMarshaler lineMarshaler =
+        new LineMarshaler(lineData.getFunctionIndex(), lineData.getLine(), lineData.getColumn());
+    return lineMarshaler;
```

Comment:
This may apply in other places as well.
```suggestion
    return new LineMarshaler(lineData.getFunctionIndex(), lineData.getLine(), lineData.getColumn());
```